docker/templates/Dockerfile.search-indexes-dump.m4

```diff
@@ -0,0 +1,28 @@
+m4_include(`server_base.m4')m4_dnl
+
+install_new_xz_utils
+
+RUN chown_mb(`/home/musicbrainz/log') && \
+    chown_mb(`/home/musicbrainz/search-indexes-dumps')
+
+copy_common_mbs_files
+
+COPY \
+    docker/musicbrainz-search-indexes-dump/consul-template-search-indexes-dump.conf \
+    /etc/
+
+COPY \
+    docker/musicbrainz-search-indexes-dump/consul-template.service \
+    /etc/service/consul-template/run
+RUN chmod 755 /etc/service/consul-template/run
+
+COPY docker/musicbrainz-search-indexes-dump/crontab /var/spool/cron/crontabs/musicbrainz
+
+RUN chown musicbrainz:musicbrainz /var/spool/cron/crontabs/musicbrainz && \
+    chmod 600 /var/spool/cron/crontabs/musicbrainz
+
+COPY docker/scripts/start_musicbrainz_server.sh /usr/local/bin/
```

Comment:
Does the container require plackup running? If not I'd guess this isn't needed

Reply:
Indeed, it does not. Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/4cef7200f0d77a0df059af5e94dc752ab34afee9